components/addFileButton/index.test.jsx

```diff
@@ -231,15 +255,24 @@ describe("handleFileSelect", () => {
   });
 
   test("it returns a handler that uploads a file", async () => {
-    await handler({ target: { files: [file] } });
+    handler({ target: { files: [file] } });
```

Comment:
A bit skeptic to the handler _not_ being asynchronous, but the code checks out. Just to clean up a bit more, you can remove the async from the test as well.